rpcs3/Emu/RSX/gcm_enums.h

```diff
@@ -992,8 +992,35 @@ enum
 
 enum Method
 {
+	/*
```

Comment:
It may be true, but they are from SDK. It will be impossible to find them (all CELL_* names are searchable).